homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
This should not be overridden in the platforms. It's only for the parent class. The attributes will be collected from existing properties, so just make sure the corresponding properties return correctly.